packages/react-components/react-infolabel-preview/src/components/InfoLabel/InfoLabel.cy.tsx

```diff
@@ -0,0 +1,82 @@
+/// <reference types="cypress-real-events" />
+
+import * as React from 'react';
+import { mount as mountBase } from '@cypress/react';
+import { FluentProvider } from '@fluentui/react-provider';
+import { teamsLightTheme } from '@fluentui/react-theme';
+import { InfoLabel } from '@fluentui/react-infolabel-preview';
```

Comment:
All imports from `@fluentui/react-infobutton` have been update to `@fluentui/react-infolabel-preview`